nonprofit-board-management.php

```diff
@@ -5,7 +5,7 @@
 Domain Path: /languages
 Plugin URI: http://wiredimpact.com/nonprofit-plugins/nonprofit-board-management/?utm_source=wordpress_admin&utm_medium=plugins_page&utm_campaign=nonprofit_board_management
 Description: A simple, free way to manage your nonprofit’s board.
-Version: 1.1.14
+Version: 1.1.15
 Author: Wired Impact
 Author URI: http://wiredimpact.com/?utm_source=wordpress_admin&utm_medium=plugins_page&utm_campaign=nonprofit_board_management
```

Comment:
Can we update this URL to use https:// instead of http://? Is it possible to do that in any other places it appears in the plugin?

Reply:
Yep, most definitely.